pkgs/servers/monitoring/prometheus/smartctl-exporter/default.nix

```
@@ -1,36 +1,17 @@
-{ lib
-, fetchFromGitHub
-, fetchpatch
-, buildGoModule
-}:
+{ lib, fetchFromGitHub, fetchpatch, buildGoModule }:
 
 buildGoModule rec {
   pname = "smartctl_exporter";
-  version = "unstable-2020-11-14";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
     owner = "prometheus-community";
     repo = pname;
-    rev = "e27581d56ad80340fb076d3ce22cef337ed76679";
-    sha256 = "sha256-iWaFDjVLBIAA9zGe0utbuvmEdA3R5lge0iCh3j2JfE8=";
+    rev = "v0.9.0";
+    sha256 = "sha256-z8z/A3BmaOSlS6/V0bEyVaUwV96pa7HM/0jad81394Q=";
```

```suggestion
    hash = "sha256-z8z/A3BmaOSlS6/V0bEyVaUwV96pa7HM/0jad81394Q=";
```

I prefer using hash these days.